### PR TITLE
A few reviews for 2014 season

### LINKED ISSUE #890: 14-15 Reviews and Impact Interviews (term 1)
These need adding at somepoint.

- [x] http://www.impactnottingham.com/2014/10/the-infant-nottingham-new-theatre/
- [x] http://www.impactnottingham.com/2014/11/punk-rock-nottingham-new-theatre/
- [x] http://www.impactnottingham.com/2014/11/bent-nottingham-new-theatre/
- [x] http://www.impactnottingham.com/2014/11/nottingham-fringe-review-infirmity/
- [x] http://www.impactnottingham.com/2014/11/our-countrys-good-nottingham-new-theatre/
- [x] http://www.impactnottingham.com/2014/12/whats-cooking-nottingham-new-theatre/
- [x] http://www.impactnottingham.com/2014/11/40-nottingham-new-theatre/
- [x] http://www.impactnottingham.com/2014/12/the-ritual-slaughter-of-gorge-mastromas-nottingham-new-theatre/
- [x] http://www.impactnottingham.com/2014/12/king-lear-nottingham-new-theatre/
- [x] http://www.impactnottingham.com/2014/12/the-adventures-of-anne-marie-de-bourbon-nottingham-new-theatre/

### PR DESCRIPTION
**Pull request by nhyty**
To close #890 eventually.

### CODE REVIEW

**Comment by nhyty** [comment]
@wjdp, why does the build fail?

**Comment by wjdp** [comment]
Hmm, something to do with infinity. Will have to have a proper look later.

**Comment by wjdp** [comment]
Is due to the logic for review stars happening in the template/Liquid #843. This also prevents us upgrading to the next Jekyll version 😢 so is worth doing quickly. I'll start work on a fix.

File: _shows/14_15/the_infant.md [hunk 1]
```diff
@@ -66,6 +66,18 @@ assets:
...
+
+links:
+  - type: Review
+    href: http://www.impactnottingham.com/2014/10/the-infant-nottingham-new-theatre/
+    snapshot: bO3d8
+    publisher: Impact Nottingham 
+    author: Henry Stanley
+    title: "The Infant @ Nottingham New Theatre"
+    date: 2014-10-30
+    stars: 4
```
Comment (wjdp) [inline _shows/14_15/the_infant.md]: Please change to `rating: 4/5` re. changes coming in from #900

File: _shows/14_15/bent.md [hunk 2]
```diff
@@ -77,6 +77,17 @@ assets:
...
 
+links:
+  - type: Review
+    href: http://www.impactnottingham.com/2014/11/bent-nottingham-new-theatre/
+    snapshot: SSTKV
+    publisher: Impact Nottingham 
+    author: Olivia Rook
+    title: "Bent @ Nottingham New Theatre"
+    date: 2014-11-20
+    rating: 9
```
Comment (wjdp) [inline _shows/14_15/bent.md]: This is the cause of the error, rating needs to be in format `x/of_y`. In this case should be `9/10`.

File: _shows/14_15/the_infant.md (hunk 1 above)
Comment (BenWoodford76) [reply _shows/14_15/the_infant.md]: Does this need to happen on all previous reviews as well or just on new additions?

Comment (nhyty) [reply _shows/14_15/the_infant.md]: Will has done it all in that branch 😄

Comment (wjdp) [reply _shows/14_15/the_infant.md]: Big ol find & replace jobbby

**Pull request merged**